tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/completion-tags-validator.js

```diff
@@ -0,0 +1,49 @@
+import { CleanText } from "../../../helpers/text-helpers";
+import { getSubmissionTimeText } from "../dpv-helpers/time-helpers";
+
+
+export const selfAssessmentPageNewRecommendation = (section, maturity) => {
+    const time = new Date();
+    const timePlusOneMinute = new Date(time).setMinutes(time.getMinutes() + 1);
+    const lateTime = new Date(timePlusOneMinute);
+
+    const matchingRecommendation = section.recommendation.intros.find(
+        (recommendation) => recommendation.maturity == maturity
+    );
+
+    if (!matchingRecommendation)
+        throw new Error(
```

Comment:
Assuming your intention is indeed to mark this test as failed, something like `expect(matchingRecommendation).to.be.ok` might be better. YMMV

Reply:
Tried it, the error message isn't very useful so will leave it as is